attributes/default.rb

```diff
@@ -263,115 +256,95 @@
+                                             httpd boost-devel redhat-lsb mlocate lvm2 mpich-devel R atlas-devel
+                                             blas-devel fftw-devel libffi-devel openssl-devel dkms mariadb-devel libedit-devel
+                                             libical-devel postgresql-devel postgresql-server sendmail libxml2-devel libglvnd-devel
+                                             mdadm python python-pip libssh2-devel libgcrypt-devel libevent-devel glibc-static bind-utils
+                                             iproute NetworkManager-config-routing-rules python3 python3-pip]
     if node['platform_version'].to_i >= 8
       # Do not install unversioned python
-      default['cfncluster']['base_packages'].delete('python')
-      default['cfncluster']['base_packages'].delete('python-pip')
-      # iptables used in configure-pat.sh
+      default['cluster']['base_packages'].delete('python')
+      default['cluster']['base_packages'].delete('python-pip')
       # gdisk required for FSx
       # environment-modules required for IntelMPI
-      # libtirpc and libtirpc-devel required for SGE
       # cryptsetup used for ephemeral drive encryption
-      default['cfncluster']['base_packages'].push(%w[iptables gdisk environment-modules libtirpc libtirpc-devel cryptsetup])
+      default['cluster']['base_packages'].push(%w[gdisk environment-modules cryptsetup])
     end
 
-    default['cfncluster']['rhel']['extra_repo'] = 'rhui-REGION-rhel-server-optional'
+    default['cluster']['rhel']['extra_repo'] = 'rhui-REGION-rhel-server-optional'
 
   when 'amazon'
-    default['cfncluster']['base_packages'] = %w[vim ksh tcsh zsh openssl-devel ncurses-devel pam-devel net-tools openmotif-devel
-                                                libXmu-devel hwloc-devel libdb-devel tcl-devel automake autoconf pyparted libtool
-                                                httpd boost-devel system-lsb mlocate atlas-devel fftw-devel glibc-static iproute
-                                                libffi-devel dkms mysql-devel libedit-devel postgresql-devel postgresql-server
-                                                sendmail cmake byacc libglvnd-devel mdadm libgcrypt-devel libevent-devel
-                                                libxml2-devel perl-devel dpkg-dev tar gzip bison flex gcc gcc-c++ patch
-                                                rpm-build rpm-sign system-rpm-config cscope ctags diffstat doxygen elfutils
-                                                gcc-gfortran git indent intltool patchutils rcs subversion swig systemtap curl
-                                                jq wget python-pip NetworkManager-config-routing-rules libibverbs-utils
-                                                librdmacm-utils python3 python3-pip]
+    default['cluster']['base_packages'] = %w[vim ksh tcsh zsh openssl-devel ncurses-devel pam-devel net-tools openmotif-devel
```

Comment:
I'm wondering if there are packages we can remove in pcluster3, for example:
* can we only install python3?
* we should really remove dpkg on non debian systems
* are there packages that were only required by SGE/Torque or Ganglia? (probably apache2?)

Reply:
If you agree I'll do it in a next patch.